docs/source/methods/avas.rst

```diff
@@ -168,6 +168,197 @@ We then use this guess of active orbitals to compute the CASCI energy:::
+
+Take the formaldehyde example above.
+We now reorient it such that the plane normal points at (1.0, 1.0, 1.0).
+The same AVAS orbitals can be obtained using the following input: ::
+
+  molecule H2CO{
+  0 1
+  C        0.346146295209737    0.126698337466632   -0.472844632676369
+  O       -0.346053592352928   -0.126664405871036    0.472717998223964
+  H        1.227335215970831   -0.489581944167415   -0.737753271803415
+  H        0.143281782803090    0.991262584630455   -1.134544367433545
+  noreorient
+  }
+
+  set forte {
+    subspace           ["C(2p)", "O(2p)"]  # must include all p orbitals!
+    subspace_pi_planes [["C", "O", "H"]]   # only one plane, defined by all C, O and H atoms
+    avas               true
+    avas_diagonalize   true
+    avas_sigma         1.0
+  }
+
+If the :code:`SUBSPACE_PI_PLANES` option is not empty, the code will prune the subspace p orbitals
+and linearly combine them to one p orbital per atom in the direction perpendicular to the plane.
+In the above example, we will end up with only two p orbitals perpendicular to the molecular plane.
+If :code:`SUBSPACE_PI_PLANES` is empty, we would have had 6 p orbitals in the subspace.
+
+.. note::
+    It is very important to include all p orbitals in :code:`SUBSPACE`.
+    Otherwise, the code will follow the directions given by :code:`SUBSPACE`.
+
+.. tip::
+    The code is flexible enough to treat double active spaces (e.g., double-π or double-d-shell).
+    For example, the double-π active space of formaldehyde can be obtained via ::
+
+      set forte {
+        minao_basis        double-shell
+        subspace           ["C(2p)","C(3p)","O(2p)","O(3p)"]
+        subspace_pi_planes [["C","O","H"]]
+        avas               true
+        avas_diagonalize   true
+        avas_cutoff        0.5
+      }
+
+    You need to prepare your own MINAO basis (here I prepare a basis called "double-shell.gbs").
+    This MINAO basis should include at least the 2p and 3p orbitals of C and O,
+    which can be grabbed from the cc-pVTZ or ANO-RCC-VTZP basis sets.
+
+For a more realistic example, consider the following iron porphyrin related molecule:
+
+.. image:: images/FeP.png
+    :width: 600
+    :align: center
+    :alt: An iron porphyrin complex.
+
+This molecule contains two π systems, namely, porphyrin and imidazole.
+Also, the porphyrin is not a perfect plane anymore.
+The following input snippet selects all 3d orbitals of Fe, 3p orbitals of S,
+and all 'pz' orbitals of porphyrin and imidazole rings. ::
+
+  set forte {
+    avas                true
+    avas_diagonalize    true
+    avas_cutoff         0.5
+    minao_basis         cc-pvtz-minao
+    subspace            ["Fe(3d)","C6-25(2p)","N(2p)","S(3p)","C1-3(2p)"]
+    subspace_pi_planes  [["Fe","C6-25","N3-6"], ["N1-2","C1-3"]]
+  }
+
+The AVAS output selects exactly 37 orbitals we wanted. ::
+
+  Sum of eigenvalues: 36.98863914
+  AVAS covers 96.59% of the subspace.
```

Comment:
Does this come from the overlap matrix? If so SVD would be good to compute and get the average of the singular values

Reply:
This is the percentage of AVAS selected orbitals over the total subspace orbitals. The SVD values are actually printed afterwards.